.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @smartcontracts
+*       @K-Ho
```

Comment:
This should be done in a separate PR so it can be merged quicker. Also if @K-Ho is a code owner here, he should be assigned as the reviewer of the PRs. @K-Ho do you have the capacity to be reviewing all tutorials?

Reply:
OK, I created a separate PR just for this change. Here is the URL: https://github.com/ethereum-optimism/optimism-tutorial/pull/72